ambry-protocol/src/main/java/com.github.ambry.protocol/AdminRequest.java

```diff
@@ -70,7 +66,7 @@ public static AdminRequest readFrom(DataInputStream stream, ClusterMap clusterMa
    * @param clientId the ID of the client.
    */
   public AdminRequest(AdminRequestOrResponseType type, PartitionId partitionId, int correlationId, String clientId) {
-    super(RequestOrResponseType.AdminRequest, ADMIN_REQUEST_VERSION_V1, correlationId, clientId);
+    super(RequestOrResponseType.AdminRequest, ADMIN_REQUEST_VERSION_V2, correlationId, clientId);
```

Comment:
Do you think it would make sense to introduce a `currentVersion` across the entire class, so you do not need to update each individual occurrence?